herval/yahoo-finance
# [WIP] V3.0.0-alpha.1

This is the first cut getting this gem working again.  

Changes

* Code extracted into modules Quotes, Historical, Splits, Symbols
* Test/Unit moved over to Minitest
* Tests extracted out my module
* Add Byebug for debugging
* Add Yard for documentation
* Change Quotes module to use the V7 Api
* Fix Historical per https://github.com/herval/yahoo-finance/pull/53
* Switch http client to https://github.com/nahi/httpclient

TODO / Questions

* Dividends - Not sure how this was ever working.  I don't see a period that related to :dividends_only
* Splits is not working . Need to figure out a new endpoint for that.
* N/A as nil still needs some work
* Documentation - This isn't a CSV endpoint so some of the concepts no longer exist such as column selection.
